src/main/groovy/life/qbic/portal/sampletracking/app/SampleTrackingPortletController.groovy

```diff
@@ -42,4 +44,10 @@ class SampleTrackingPortletController implements PortletController {
     void clearSelection() {
         this.sampleListModification.clearSelection()
     }
-}
+
+    @Override
+    void updateSamples(String sampleIds, Location desiredLocation, Status desiredStatus) {
```

Comment:
If we decide to change the interface method this method would of course be changed as well.

Reply:
```suggestion
    void updateSamples(List<String> sampleIds, Location desiredLocation, Status desiredStatus) {
```